aws_cloudfront_distribution.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_cloudfront_distribution" "s3-distribution-TLS-v1" {
     }
   }
 
-  enabled             = true
+  enabled = true
 
   default_cache_behavior {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Use encrypted connection between CloudFront and origin server **
 
**Vulnerability**
Use encrypted connection between CloudFront and origin server 
 
**Remediation**
Enforce HTTPS-only traffic between a CloudFront distribution and the origin. Note: This rule runs on all the origins except S3 Buckets

source: accurics-